from unittest import TestCase


class Test(TestCase):
    def test_has_argument(self):
        self.fail()
